nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/component-state/component-state.component.html

```diff
@@ -0,0 +1,86 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="component-state-dialog" tabindex="0">
+    <h2 mat-dialog-title>Component State</h2>
+    <mat-dialog-content>
+        <div class="flex flex-col justify-between gap-y-5">
+            <div class="flex flex-col" *ngIf="componentName$ | async; let componentName">
+                <div>Name</div>
+                <div class="value">{{ componentName }}</div>
+            </div>
+            <div class="flex flex-col">
+                <div>Description</div>
+                <div class="value">
+                    {{ stateDescription }}
+                </div>
+            </div>
+            <div class="listing-table">
+                <form [formGroup]="filterForm" class="flex flex-col gap-y-2">
+                    <div class="value">Displaying {{ filteredEntries }} of {{ totalEntries }}</div>
+                    <div class="flex justify-between items-center">
+                        <mat-form-field>
+                            <mat-label>Filter</mat-label>
+                            <input matInput type="text" class="small" formControlName="filterTerm" />
+                        </mat-form-field>
+                        <ng-container *ngIf="{ value: (canClear$ | async)! } as canClear">
+                            <div *ngIf="canClear.value && totalEntries > 0">
+                                <a (click)="clearState()">Clear state</a>
+                            </div>
```

Comment:
NiFi actually shows the clear link but disabled rather than hiding it. I think I prefer hiding it as well, just wanted to make sure this was a conscious decision to deviate from NiFi.

Reply:
This was a conscious decision.